drivers/boston_project_scorecard/app/models/boston_project_scorecard/financial_performance.rb

```diff
@@ -0,0 +1,116 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module BostonProjectScorecard
+  module FinancialPerformance
+    extend ActiveSupport::Concern
+    included do
+      def project_type_value
+        case project_type
+        when 3
+          'PSH'
+        when 9, 10
+          'PH'
+        when 13
+          'RRH'
+        else
+          'Other'
+        end
+      end
+
+      def project_type_score
+        return 6 if project_type == 3 # PSH
+        return unless project_type.in?(9, 10, 13)
```

Comment:
should this return the actual project type?

Reply:
maybe just a comment about why certain project types are being used and not others?

Reply:
The spec says:
6: PSH
3: RRH/Joint TH-PH
So, I'm returning null for the others